platform/view/services/comm/host/rest/websocket/multiplexed_provider.go

```diff
@@ -186,15 +186,15 @@ func (c *multiplexedClientConn) readIncoming() {
 			sc.reads <- streamEOF
 		}
 		err := c.Conn.Close()
-		logger.Infof("Client connection closed: %v", err)
+		logger.Debugf("client connection closed: %v", err)
 	}()
 	var mm MultiplexedMessage
 	for {
 		//c.writeMu.Lock()
 		err := c.ReadJSON(&mm)
 		//c.writeMu.Unlock()
 		if err != nil {
-			logger.Warnf("Client connection errored: %v", err)
+			logger.Debugf("Client connection errored: %v", err)
```

Comment:
for consistency, maybe all of them should start with lowercase? (also on line 207 and 209)

Reply:
done